src/main/webapp/i18n/de/assessment.json

```diff
@@ -42,7 +42,8 @@
                 "saveSuccessful": "Deine Bewertung wurde erfolgreich gespeichert!",
                 "saveFailed": "Das Speichern deiner Bewertung ist fehlgeschlagen!",
                 "updateAfterComplaintSuccessful": "Die Bewertung wurde erfolgreich aktualisiert.",
-                "updateAfterComplaintFailed": "Das Aktualisieren der Bewertung ist fehlgeschlagen."
+                "updateAfterComplaintFailed": "Das Aktualisieren der Bewertung ist fehlgeschlagen.",
+                "removeGradingInstructionLink": "Möchten Sie den Link zum Benotungshinweis entfernen?"
```

Comment:
I just started testing again, one comment beforehand: we do not use the polite/formal "Sie-Form". So the translation would be:
Möchtest du den Link zum Benotungshinweis entfernen?